src/app/components/surveillance/complaints/complaint-view.jsx

```diff
@@ -15,11 +15,35 @@ import {
 import theme from '../../../themes/theme';
 import { getAngularService } from '../../../services/angular-react-helper';
 import { complaint as complaintPropType } from '../../../shared/prop-types';
+import { CardHeader } from '@material-ui/core';
```

Comment:
We generally put the imports that are coming from external to our system at the top, and the ones that are from internal below them, so this line ought to be up in the section from lines 1 to 13. More importantly, we're already importing stuff from `material-ui/core` in a group, so this one should be added to that section (alphabetically)